mozetl/taar/taar_amowhitelist.py

```diff
@@ -64,12 +64,24 @@ def transform(self, json_data):
 
         new_data = {}
         for guid in json_data.keys():
-            rating = json_data[guid]['ratings']['average']
-            create_date = parse(json_data[guid]['first_create_date']).replace(tzinfo=None)
+            addon_data = json_data[guid]
+
             if guid == 'pioneer-opt-in@mozilla.org':
                 # Firefox Pioneer is explicitly excluded
                 continue
 
+            current_version_files = addon_data.get('current_version', {}).get('files', [])
+            if len(current_version_files) == 0:
+                # Only allow webextensions
+                continue
+            else:
```

Comment:
Is this `else` really needed? If we stumble upon the previous case, then we `continue`. I think we can just drop the `else` to reduce indentation as per our [style guide](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style) (see "General practices").